app/assets/scripts/components/map/legend.js

```diff
@@ -1,14 +1,16 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import { useHistory } from 'react-router-dom';
 import { Dropdown } from 'openaq-design-system';
 import c from 'classnames';
 
 import { generateLegendStops } from '../../utils/colors';
```

Comment:
not exactly sure the issue, but seeing NaN produced on several parameters. This must be coming from an undefined parameterMax? can we get this from the API?

Also after clicking on a view params, I see too many updates on the legend. on initial load of this param, the legend looks correct
![image](https://user-images.githubusercontent.com/19439427/102532011-4f3fde80-4071-11eb-9887-06290af06a26.png)